src/main/java/de/tum/in/test/api/ast/model/UnwantedNode.java

```diff
@@ -0,0 +1,137 @@
+package de.tum.in.test.api.ast.model;
+
+import java.nio.file.Path;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import com.github.javaparser.ast.Node;
+
+/**
+ * Stores all unwanted nodes of an abstract syntax tree of a Java-file
+ */
+public class UnwantedNode {
+
+	private final String unwantedNodeName;
+	private final List<NodePosition> unwantedNodePositions;
+
+	public UnwantedNode(JavaFile javaFile, String unwantedNodeName, Class<? extends Node> nodeDefinedAsUnwanted) {
+		this.unwantedNodeName = unwantedNodeName;
+		this.unwantedNodePositions = javaFile.getJavaFileAST().findAll(nodeDefinedAsUnwanted).stream()
+				.map(NodePosition::getPositionOf).collect(Collectors.toList());
+		this.unwantedNodePositions
+				.sort(Comparator.comparingInt(NodePosition::getBeginLine).thenComparing(NodePosition::getBeginColumn));
```

Comment:
Consider making NodePosition comparable, if it is a frequent sorting key.

Reply:
[Implemented part of the new feedback](https://github.com/ls1intum/Ares/pull/248/commits/ead5d75f0814dbd3738c57b058de8759b980e46d): Realised the interface `Comparable<NodePosition>` in the class `NodePosition` and subsequently updated all sorting-method calls.